core/templates/app.constants.ts

```diff
@@ -50,6 +50,9 @@ export const AppConstants = {
   EXPLORATION_DATA_URL_TEMPLATE: '/explorehandler/init/<exploration_id>',
   EXPLORATION_VERSION_DATA_URL_TEMPLATE: (
     '/explorehandler/init/<exploration_id>?v=<version>'),
+  ENTITY_TRANSLATIONS_HANDLER_URL_TEMPLATE: (
+    '/entity_translations_handler/<entity_type>/<entity_id>/<entity_version>/' +
+    '<language_code>'),
```

Comment:
This can be put in _constants.ts_ and then doesn't need to be added here or and can be used in _main.py_

Reply:
This is the general pattern we follow i.e, define the constants in both the backend and frontend. Through I agree moving it to constants.ts would be better.

I checked constants.ts and I don't see any URL mentioned there, should we follow the existing pattern? (I can add it in the constants.ts if you like!)

Reply:
Hmm, good point, yeah I think these should just be moved together into _constants.ts_

Reply:
But not sure if now, up to you. I think we even have an issue for unifying the URLs from BE and FE.